torch/csrc/jit/passes/python_print.cpp

```diff
@@ -193,7 +193,7 @@ struct PythonPrintPass {
   // constants are written to this table, and given then named CONSTANTS.cN
   // where N is the index into this table.
 
-  std::vector<at::Tensor> tensor_constants;
+  std::vector<at::Tensor>& tensor_table_;
```

Comment:
Nit: shall we turn tensor_table_ to a pointer type (i.e., std::vector<at::Tensor>*)?
Having a reference type as a member looks not very appropriate to me.

Reply:
I don't see a big reason to change it, we already use an ostream& in this class as well. The class is really just being used a a closure that holds common fields across the recursive set of export methods.